crates/sui-indexer/src/models/objects.rs

```diff
@@ -199,7 +199,19 @@ impl TryFrom<Object> for sui_types::object::Object {
                     .map(|NamedBcsBytes(name, bytes)| (name, bytes))
                     .collect();
                 // Ok to unwrap, package size is safe guarded by the full node, we are not limiting size when reading back from DB.
-                let package = MovePackage::new(object_id, version, &modules, u64::MAX).unwrap();
+                let package = MovePackage::new(
+                    object_id,
+                    version,
+                    modules,
+                    u64::MAX,
+                    // TODO: these represent internal data needed for Move code execution and as
+                    // long as this MovePackage does not find its way to the Move adapter (which is
+                    // the assumption here) they can remain uninitialized though we could consider
+                    // storing them in the database and properly initializing here for completeness
```

Comment:
cc @gegaowp, @666lcz  -- (not urgent) it'll probably be helpful for other reasons to have the indexer know what a package's true transitive dependencies are.  Is this something you can add?

Reply:
Thanks for the heads-up and sry for the late reply!
tbh I was not aware of this, but we did plan to split package object & other objects into different tables.
I will keep this in mina and ask your guidance to add that in the new package table.